src/utils/trusts.ts

```diff
@@ -52,19 +55,27 @@ const checkEntityReviewRequiresTrusts = (appData: ApplicationData | undefined):
  * @param appData Application Data
  * @returns string URL to go to when starting the trust journey
  */
-const getTrustLandingUrl = (appData: ApplicationData): string => {
+const getTrustLandingUrl = (appData: ApplicationData, req?: Request): string => { //  TODO MAKE REQ MANDATORY
```

Comment:
We need to add a JIRA sub-task somewhere, to address the TODO comment present here and highlighted in the code smell. I'll see if I can find a suitable story for it to live under.

Reply:
Could either add it to the final trust story (not sure what that is) or could add it to this story - https://companieshouse.atlassian.net/browse/ROE-1996. When we come to remove the Redis dependency, there should, in theory, be no more invocations of this method with just one parameter.

Reply:
adde this task - https://companieshouse.atlassian.net/browse/ROE-2488